doc/nrf/protocols/matter/end_product/security.rst

```diff
@@ -7,14 +7,44 @@ Security
    :local:
    :depth: 3
 
-Nordic Matter samples leverage security features supported in the |NCS| that can be divided into three major categories:
+Nordic Matter samples leverage :ref:`security` features supported in the |NCS| that can be divided into four major categories:
 
+* Secure processing environment
 * Cryptography
 * Trusted storage
 * Securing production devices
 
 In the following sections you will learn more details about each listed category.
 
+Secure processing environment
+*****************************
+
+Depending on the board, Matter samples can use a secure processing environment.
+
+nRF54L with Trusted Firmware-M (TF-M)
+=====================================
+
+On the nRF54L SoC, Matter samples support :ref:`app_boards_spe_nspe` with Trusted Firmware-M (TF-M).
+All cryptographic operations within the Matter stack are performed by utilizing the `Platform Security Architecture (PSA)`_ API and executed in the secure TF-M environment.
+The secure materials like Matter Session keys, DAC private key and other keys, are stored in the TF-M secure storage utilizing the :ref:`tfm_encrypted_its` module.
+Matter samples utilize the full TF-M library, so you cannot use the :ref:`tfm_minimal_build` version of TF-M.
+
+To build a Matter sample with the TF-M support add the ``/ns`` variant to the board target.
+For example, use the following command for the nRF54L15 SoC:
+
+.. code-block:: console
+
+   west build -b nrf54l15dk/nrf54l15/cpuapp/ns
```

Comment:
This is a very basic information that IMO should be covered by a cross-link, as suggested in my first review.
If there were additional parameters in the command, providing it here has some value, but duplicating the very basic build command and ignoring VSC extension, when the Building page covers both, is in my opinion a detriment.

Reply:
```suggestion
To build a Matter sample with the TF-M support, :ref:`build <building>` for the :ref:`board target <app_boards_names>` with the ``/ns`` variant.
```
As suggested in  https://github.com/nrfconnect/sdk-nrf/pull/18321#discussion_r1820995047